gprofiler/profilers/python.py

```diff
@@ -82,6 +85,48 @@ def _add_versions_to_stacks(
     return result
 
 
+class PythonMetadta(ApplicationMetadata):
+    _PYTHON_VERSION_TIMEOUT = 3
+
+    @classmethod
+    def _get_python_version(cls, process: Process, stop_event: Event) -> str:
+        python_path = f"/proc/{get_process_nspid(process.pid)}/exe"
```

Comment:
Regarding our conversation - how do you know that this is `python` exe?

Reply:
I don't, this is TODO code, can't get the version if not `python` binary here :/ We'll need to figure it out.

Reply:
How PyPerf and py-spy know that a process is Python? Can't we rely on them?
And at least we can validate that the result from running the process makes sense, i.e. that it looks like "Python x.y.z".

Reply:
MVP - I'll start with checking `basename(exe).startwith("python")`.

@YishaiZinkin - gProfiler discovers Python processes using the `DETECTED_PYTHON_PROCESSES_REGEX` regex. PyPerf is a bit simplified and only checks for python/libpython.